src/ncp/ncp_openthread.cpp

```diff
@@ -87,12 +87,41 @@ ControllerOpenThread::~ControllerOpenThread(void)
     assert(mInstance == nullptr);
 }
 
-void ControllerOpenThread::Init(void)
+otbrLogLevel ControllerOpenThread::ConvertLogLevelOtToOtbr(otLogLevel aLogLevel)
```

Comment:
The name is well self-explained, however it seems a bit redundant? how about changing it as below?

```
otbrLogLevel ControllerOpenThread::ConvertLogLevelToOtbr(otLogLevel aLogLevel)
```
or
```
otbrLogLevel ControllerOpenThread::ConvertToOtbrLogLevel(otLogLevel aLogLevel)
```

Reply:
ConvertToOtbrLogLevel sgtm.